atomicfu/build.gradle.kts

```diff
@@ -86,6 +90,7 @@ kotlin {
 
     sourceSets {
         commonMain.dependencies {
+            implementation("org.jetbrains.kotlinx:kotlinx-datetime:0.6.2")
```

Comment:
This dependency is too heavy to be used here, and it also doesn't seem like it's used.

Reply:
Oh my bad!